troposphere/templates/login.html

```diff
@@ -84,7 +83,7 @@
                 {% csrf_token %}
                 {% if disable_login == False %}
                 <a class="submitButton" href="{{ BASE_URL }}/login">
-                        Log in with your iPlant ID
+                        Log in with your {{ ORG_NAME }} ID
```

Comment:
The issue you could say with this is that you're logging into Atmosphere with your CyVerse ID. You're not really logging into "CyVerse" - other than in the situation of Jetstream (perhaps). 

So here, for CyVerse, I think "Log in CyVerse" is a bit odd some there really isn't a CyVerse to log into - but one of the application built by CyVerse. 

Or maybe I'm over thinking this...